woocommerce/woocommerce-android
# Use Automattic-Tracks-Android library from S3 Maven

### Description


This PR updates the `Automattic-Tracks-Android` library dependency so it'll be fetched from S3 Maven instead of Jitpack.

### Testing instructions


[There are no functionality changes between `2.1.0` & `2.2.0` versions of the library within the tracks module](https://github.com/Automattic/Automattic-Tracks-Android/releases/tag/2.2.0), so as long as the CI is green, I think we should be fine. Making sure the installable build works and checking if the events are properly reported to tracks would be the next steps of testing if the reviewer wants to be extra sure about the change.

**Edit:** I verified the events are being tracked from the `Tracks Live View`.
